### PR TITLE
API 2.0 swagger validator

### PR DESCRIPTION
**Pull request by dalebremner**
@RackHD/corecommitters @zyoung51 @srinia6 @geoff-reid @abildw

### CODE REVIEW

File: static/schemas/2.0/Node.2.0.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                "name": {
+                    "description": "Name of the node",
+                    "type": "string"
+                },
+                "type": {
+                    "description": "Type of node",
+                    "type": "string",
+                    "enum": ["compute", "switch", "dae", "pdu", "mgmt", "enclosure"]
+                },
+                "obmSettings": {
```
Comment (zyoung51) [inline static/schemas/2.0/Node.2.0.json]: This is a first pass and the obmSettings/snmpSettings/bootSettings will have their own detailed schemas later?

File: static/schemas/2.0/Node.2.0.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "Name of the node",
+                    "type": "string"
+                },
+                "type": {
+                    "description": "Type of node",
+                    "type": "string",
+                    "enum": ["compute", "switch", "dae", "pdu", "mgmt", "enclosure"]
```
Comment (zyoung51) [inline static/schemas/2.0/Node.2.0.json]: @jlongever is adding redfish types here as well.

File: static/schemas/2.0/Node.2.0.json (hunk 1 above)
Comment (dalebremner) [inline static/schemas/2.0/Node.2.0.json]: @zyoung51 Yes, just a first pass schema...

**Comment by zyoung51** [comment]
+1

File: static/schemas/2.0/Node.2.0.json (hunk 2 above)
Comment (jlongever) [inline static/schemas/2.0/Node.2.0.json]: Actually,  I only added Redfish type support for OBM settings.   The below schema definition for OBM settings will require redfish as a type on the subsequent pass of the schemas.

**Comment by brianparry** [comment]
:+1:

**Comment by srinia6** [comment]
+1

**Pull request merged**